apps/google-analytics-4/lambda/src/services/googleApiUtils.ts

```diff
@@ -0,0 +1,147 @@
+import { ERROR_TYPE_MAP } from '../errors/apiError';
```

Comment:
This contains the logic of mapping Google Statuses to our own custom defined statuses